src/src/rx_main.cpp

```diff
@@ -1012,52 +1012,55 @@ void UpdateModelMatch(uint8_t model)
  **/
 void MspReceiveComplete()
 {
-    if (MspData[7] == MSP_SET_RX_CONFIG && MspData[8] == MSP_ELRS_MODEL_ID)
-    {
-        UpdateModelMatch(MspData[9]);
-    }
-    else if (MspData[0] == MSP_ELRS_SET_RX_WIFI_MODE)
-    {
+    switch (MspData[0]) {
```

Comment:
Pedantic: Styling of the curly brace here is inconsistent with the rest of the repo. Curlys should be on a new line.

Reply:
Done.